### ISSUE TITLE
Add-on no longer works with Firefox version 35.0

### ISSUE THREAD

**Issue by X-Looper** [IsSuEat/open-livestreamer-firefox-addon#9]
Hello,

I noticed a few days ago when Firefox updated to version 35.0 that this add-on stopped working, it's no longer possible to launch Livestreamer on twitch.tv to watch the Streams through VLC and it would be very much appreciated if we could get an update to fix this issue, it's a really great add-on and it makes watching streams so much better, thanks!

/Regards X-Looper

**Comment by baizon** [comment]
It's working for me. Did you try to launch livestreamer from the command line?
<code>livestreamer http://www.twitch.tv/streamername best --player "mpv --no-cache --hwdec=vdpau" </code>

**Comment by X-Looper** [comment]
Hello baizon,

Yes, I did try launching a Stream from the command line and that did work. It just doesn't seem to work when I right-click on the Stream thumbnail and select "Open with Livestreamer" from the drop-down menu, when I do nothing happens and I'm not sure why.

/Regards X-Looper

**Comment by baizon** [comment]
Are your "Path to livestreamer", "Optional arguments for livestreamer" and "Steam quality" correct? 
For example, if "Stream quality" only goes to "high" and not "best", the player wont start. So check first what streaming quality is supported by the streamer.

**Comment by X-Looper** [comment]
It's on the default install path right now "/usr/bin/livestreamer" and I tried changing that to the path on my computer to the executable and when I do I get this error message "Livestreamer binary not found (or wrong binary), please specify the path in the Add-On Manager Settings"

I don't think there's any problem with the Stream Quality, I don't even get a command prompt or anything when I right-click on the Stream thumbnail and select Open with Livestreamer, nothing happens at all, no command prompt opening up and trying to launch Livestreamer.

Also, thanks for trying to help me with this, I really appreciate and would love to get this working again :-)

**Comment by baizon** [comment]
OK, so i have a few questions for better debugging:
- What OS do you have?
- Which version of livestreamer do you use?
- What are your entries in:
.- Path to livestreamer
.- Optional arguments for livestreamer 
.- Steam quality

**Comment by X-Looper** [comment]
Windows 7 Ultimate
Livestreamer version 1.11.1
Path: /usr/bin/livestreamer (it doesn't give me any error message)
Optional Arguments: None (it's completely blank)
Stream Quality: Best

I kinda have a new thought. The thing is, the last time I could run Livestreamer through Firefox was just before I accidentally hit something on my keyboard while trying to type in the name of a game in the search field on twitch.tv, I believe I actually pressed down both Shift and Ctrl on my keyboard and it opened up some window in Firefox (sadly I don't remember what it was, I closed it down just as fast as it popped up), but is it possible that there is some kind of key short-cut command in Firefox that could disable this somehow from working correctly?

Anyway, it was pretty much right after this when I tried to open up a Stream in Firefox that I noticed it wouldn't work anymore. I thought maybe it was Firefox making a silent update to version 35.0, but now I think it might've been that keyboard screw-up I did in the search-field, any ideas? :-(

/Regards X-Looper

**Comment by baizon** [comment]
That's what it looks like on my PC:
- Path to livestreamer <code>D:\livestreamer-v1.11.1\livestreamer.exe</code>
- Optional arguments for livestreamer <code>--player=D:\mpv-x86_64-latest\mpv.exe --no-cache</code>
- Steam quality <code>best</code>
You have to change the parameters, and it should work again.

**Comment by X-Looper** [comment]
You did it!

My god, I cannot believe it was so simple. I didn't know you had to write the filename at the end of the path to Livestreamer as well and now that I added \livestreamer.exe at the end of the path it works, IT WORKS!

Many thanks for helping me with this baizon, I do feel a bit stupid now, but man I'm so glad I got it working again, thanks a lot really and thanks for this great add-on as well! :-))

/Regards X-Looper
